[2024-01-31 19:22:23,410][root][INFO] - using 1 gpus
the PU data is ['KA04', 'KA15', 'KA16', 'KA22', 'KA30', 'KB23', 'KB24', 'KB27', 'KI14', 'KI17', 'KI21']
the PU data is ['KA04', 'KA15', 'KA16', 'KA22', 'KA30', 'KB23', 'KB24', 'KB27', 'KI14', 'KI17', 'KI21']
--------PU Dataset:(2, 0) TL Task---------
the input kind is fft,the model kind is ResNet18
the opt infor is {'name': 'adamw', 'lr_src': 0.001, 'lr_tar': 0.001, 'lr_scheduler': 'designed', 'weight_decay_src': 0.0001, 'weight_decay_tar': 0.0001}
the data infor is {'data_name': 'PU', 'data_path': 'D:/P_new/PU_dataset', 'TL_list': [0, 1, 2], 'input_kind': 'fft', 'norm_kind': 'mean-std', 'TL_Task': [2, 0]}
100%|██████████| 11/11 [00:00<00:00, 37.00it/s]
100%|██████████| 11/11 [00:00<00:00, 37.77it/s]
the initial accc is 71.58
layer index: 2 	 total channel: 64 	 remaining channel: 29
layer index: 8 	 total channel: 64 	 remaining channel: 22
layer index: 11 	 total channel: 64 	 remaining channel: 28
layer index: 14 	 total channel: 64 	 remaining channel: 30
layer index: 17 	 total channel: 64 	 remaining channel: 23
layer index: 21 	 total channel: 128 	 remaining channel: 62
layer index: 24 	 total channel: 128 	 remaining channel: 51
layer index: 30 	 total channel: 128 	 remaining channel: 62
layer index: 33 	 total channel: 128 	 remaining channel: 54
layer index: 37 	 total channel: 256 	 remaining channel: 113
layer index: 40 	 total channel: 256 	 remaining channel: 105
layer index: 46 	 total channel: 256 	 remaining channel: 100
layer index: 49 	 total channel: 256 	 remaining channel: 98
layer index: 53 	 total channel: 512 	 remaining channel: 211
layer index: 56 	 total channel: 512 	 remaining channel: 213
layer index: 62 	 total channel: 512 	 remaining channel: 189
layer index: 65 	 total channel: 512 	 remaining channel: 158
[2024-01-31 19:22:26,477][urllib3.connectionpool][WARNING] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
the prune ratio is 49.08 and the acc_t is 9.07
Task: [2, 0], Iter:1/50; Accuracy_S/T = 86.66%/ 45.52%; Loss= 1.57
Task: [2, 0], Iter:2/50; Accuracy_S/T = 98.66%/ 60.73%; Loss= 0.77
[2024-01-31 19:22:36,890][urllib3.connectionpool][WARNING] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [2, 0], Iter:3/50; Accuracy_S/T = 99.85%/ 64.83%; Loss= 0.64
Task: [2, 0], Iter:4/50; Accuracy_S/T = 100.00%/ 67.95%; Loss= 0.61
[2024-01-31 19:22:47,330][urllib3.connectionpool][WARNING] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [2, 0], Iter:5/50; Accuracy_S/T = 100.00%/ 70.56%; Loss= 0.59
Task: [2, 0], Iter:6/50; Accuracy_S/T = 100.00%/ 72.09%; Loss= 0.59
Task: [2, 0], Iter:7/50; Accuracy_S/T = 100.00%/ 67.44%; Loss= 0.58
Task: [2, 0], Iter:8/50; Accuracy_S/T = 100.00%/ 67.62%; Loss= 0.58
Task: [2, 0], Iter:9/50; Accuracy_S/T = 100.00%/ 67.62%; Loss= 0.57
[2024-01-31 19:23:08,177][urllib3.connectionpool][WARNING] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [2, 0], Iter:10/50; Accuracy_S/T = 100.00%/ 70.53%; Loss= 0.57
Task: [2, 0], Iter:11/50; Accuracy_S/T = 100.00%/ 69.66%; Loss= 0.57
[2024-01-31 19:23:18,609][urllib3.connectionpool][WARNING] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [2, 0], Iter:12/50; Accuracy_S/T = 100.00%/ 68.53%; Loss= 0.57
Task: [2, 0], Iter:13/50; Accuracy_S/T = 100.00%/ 69.66%; Loss= 0.57
Task: [2, 0], Iter:14/50; Accuracy_S/T = 100.00%/ 70.16%; Loss= 0.56
[2024-01-31 19:23:29,045][urllib3.connectionpool][WARNING] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [2, 0], Iter:15/50; Accuracy_S/T = 100.00%/ 69.18%; Loss= 0.57
Task: [2, 0], Iter:16/50; Accuracy_S/T = 100.00%/ 69.87%; Loss= 0.56
Task: [2, 0], Iter:17/50; Accuracy_S/T = 100.00%/ 67.99%; Loss= 0.56
Task: [2, 0], Iter:18/50; Accuracy_S/T = 100.00%/ 69.36%; Loss= 0.56
Task: [2, 0], Iter:19/50; Accuracy_S/T = 100.00%/ 68.60%; Loss= 0.56
Task: [2, 0], Iter:20/50; Accuracy_S/T = 100.00%/ 68.53%; Loss= 0.56
Task: [2, 0], Iter:21/50; Accuracy_S/T = 100.00%/ 69.33%; Loss= 0.56
Task: [2, 0], Iter:22/50; Accuracy_S/T = 100.00%/ 69.15%; Loss= 0.56
Task: [2, 0], Iter:23/50; Accuracy_S/T = 100.00%/ 70.34%; Loss= 0.56
Task: [2, 0], Iter:24/50; Accuracy_S/T = 100.00%/ 66.72%; Loss= 0.56
Task: [2, 0], Iter:25/50; Accuracy_S/T = 100.00%/ 68.89%; Loss= 0.56
Task: [2, 0], Iter:26/50; Accuracy_S/T = 100.00%/ 69.73%; Loss= 0.56
Task: [2, 0], Iter:27/50; Accuracy_S/T = 100.00%/ 68.82%; Loss= 0.56
Task: [2, 0], Iter:28/50; Accuracy_S/T = 100.00%/ 68.60%; Loss= 0.55
Task: [2, 0], Iter:29/50; Accuracy_S/T = 100.00%/ 68.42%; Loss= 0.56
Task: [2, 0], Iter:30/50; Accuracy_S/T = 100.00%/ 69.22%; Loss= 0.55
Task: [2, 0], Iter:31/50; Accuracy_S/T = 100.00%/ 69.66%; Loss= 0.55
Task: [2, 0], Iter:32/50; Accuracy_S/T = 100.00%/ 69.36%; Loss= 0.56
Task: [2, 0], Iter:33/50; Accuracy_S/T = 100.00%/ 69.44%; Loss= 0.56
Task: [2, 0], Iter:34/50; Accuracy_S/T = 100.00%/ 70.38%; Loss= 0.55
Task: [2, 0], Iter:35/50; Accuracy_S/T = 100.00%/ 69.15%; Loss= 0.56
Task: [2, 0], Iter:36/50; Accuracy_S/T = 100.00%/ 69.33%; Loss= 0.55
Task: [2, 0], Iter:37/50; Accuracy_S/T = 100.00%/ 68.86%; Loss= 0.56
Task: [2, 0], Iter:38/50; Accuracy_S/T = 100.00%/ 68.89%; Loss= 0.55
Task: [2, 0], Iter:39/50; Accuracy_S/T = 100.00%/ 68.20%; Loss= 0.55
Task: [2, 0], Iter:40/50; Accuracy_S/T = 100.00%/ 68.97%; Loss= 0.55
Task: [2, 0], Iter:41/50; Accuracy_S/T = 100.00%/ 69.62%; Loss= 0.55
Task: [2, 0], Iter:42/50; Accuracy_S/T = 100.00%/ 69.07%; Loss= 0.55
Task: [2, 0], Iter:43/50; Accuracy_S/T = 100.00%/ 68.57%; Loss= 0.55
Task: [2, 0], Iter:44/50; Accuracy_S/T = 100.00%/ 68.64%; Loss= 0.55
Task: [2, 0], Iter:45/50; Accuracy_S/T = 100.00%/ 69.07%; Loss= 0.55
Task: [2, 0], Iter:46/50; Accuracy_S/T = 100.00%/ 69.07%; Loss= 0.55
Task: [2, 0], Iter:47/50; Accuracy_S/T = 100.00%/ 69.58%; Loss= 0.55
Task: [2, 0], Iter:48/50; Accuracy_S/T = 100.00%/ 69.51%; Loss= 0.55
Task: [2, 0], Iter:49/50; Accuracy_S/T = 100.00%/ 69.55%; Loss= 0.55
Task: [2, 0], Iter:50/50; Accuracy_S/T = 100.00%/ 69.44%; Loss= 0.55